cutlass/cf.go

```diff
@@ -356,23 +358,81 @@ func (a *App) PushNoStart() error {
 	return nil
 }
 
-func (a *App) V3Push() error {
-	if err := a.PushNoStart(); err != nil {
+func (a *App) V3CreateApp() error {
+	args := []string{"v3-create-app", a.Name}
+	command := exec.Command("cf", args...)
+	command.Stdout = DefaultStdoutStderr
+	command.Stderr = DefaultStdoutStderr
+	if err := command.Run(); err != nil {
```

Comment:
Either return with a fmt.Errorf to clarify what the error is, or just return command.Run() to simplify the command.